bin/node/executor/tests/common.rs

```diff
@@ -207,10 +209,13 @@ pub fn construct_block(
 	let header = match executor_call::<NeverNativeValue, fn() -> _>(
 		env,
 		"BlockBuilder_finalize_block",
-		&[0u8;0],
+		&[0u8; 0],
 		true,
 		None,
-	).0.unwrap() {
+	)
+	.0
+	.unwrap()
+	{
```

Comment:
Creating a temp var for the result of `executor_call` would probably make this less weird.